README.md

```diff
@@ -83,13 +86,15 @@ Usage of bin/docker-quobyte-plugin:
 ### Create a volume
 
 ```
-$ docker volume create --driver quobyte --name <volumename> --opt tenant_id=<your Quobyte tenant_id>
-# Set user and group of the volume
-$ docker volume create --driver quobyte --name <volumename> --opt user=docker --opt group=docker --opt tenant_id=<your Quobyte tenant_id>
+$ docker volume create --driver quobyte --name <volumename> 
+# Set user, group and specific volume configuration for the new volume
+$ docker volume create --driver quobyte --name <volumename> --opt user=docker --opt group=docker --opt configuration_name=SSD_ONLY
```

Comment:
Should `configuration_name` default to the default configuration?

Reply:
The default configuration is already provided by the settings, the example shows using a non-default configuration_name. I'll update the comment on this example.